src/main/java/graphql/nadel/TypeDefinitionsWithRuntimeWiring.java

```diff
@@ -0,0 +1,56 @@
+package graphql.nadel;
+
+import graphql.schema.idl.TypeDefinitionRegistry;
+import graphql.schema.idl.TypeRuntimeWiring;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Stores {@link TypeDefinitionRegistry} and collection of {@link TypeRuntimeWiring}s.
+ */
+public class TypeDefinitionsWithRuntimeWiring {
+    private final TypeDefinitionRegistry typeDefinitionRegistry;
+    private final List<TypeRuntimeWiring> typeRuntimeWirings;
+
+    private TypeDefinitionsWithRuntimeWiring(TypeDefinitionRegistry typeDefinitionRegistry,
+                                             List<TypeRuntimeWiring> typeRuntimeWirings) {
+        this.typeDefinitionRegistry = typeDefinitionRegistry;
+        this.typeRuntimeWirings = typeRuntimeWirings;
+    }
+
+    public static Builder newTypeDefinitionWithRuntimeWiring() {
+        return new Builder();
+    }
+
+    public TypeDefinitionRegistry typeDefinitionRegistry() {
+        return typeDefinitionRegistry;
+    }
+
+    public List<TypeRuntimeWiring> typeRuntimeWirings() {
+        return typeRuntimeWirings;
+    }
+
+    public static class Builder {
+        private TypeDefinitionRegistry typeDefinitionRegistry = new TypeDefinitionRegistry();
+        private List<TypeRuntimeWiring> typeRuntimeWirings = new ArrayList<>();
+
+        private Builder() {
+        }
+
+        public Builder withTypeDefinitionRegistry(TypeDefinitionRegistry registry) {
+            this.typeDefinitionRegistry = Objects.requireNonNull(registry, "registry");
+            return this;
+        }
+
+        public Builder withTypeRuntimeWiring(TypeRuntimeWiring wiring) {
```

Comment:
just a side note: the naming of the builder is of: graphql-java names it differently a bit from braid and nadel :-(

Reply:
will fix